src/common/platform/process.hpp

```diff
@@ -977,11 +982,17 @@ struct GDI_SHARED_MEMORY64
 
 static_assert(offsetof(GDI_SHARED_MEMORY64, Objects) == 0x1800B0);
 
-struct CLIENT_ID64
+typedef struct _CLIENT_ID32
+{
+    ULONG UniqueProcess;
+    ULONG UniqueThread;
+} CLIENT_ID32, *PCLIENT_ID32;
+
+typedef struct _CLIENT_ID64
 {
     DWORD64 UniqueProcess;
     DWORD64 UniqueThread;
-};
+} CLIENT_ID64, *PCLIENT_ID64;
```

Comment:
don't use typedef struct here. The reason is the following: introducing types like PCLIENT_ID64 gives the impression you could use them. however, you can not. as pointer types are dependant on the compilation platform, however, for emulation, they in fact depend on the architecture of the emulated binary.